modules/monitoring-querying-metrics-for-user-defined-projects-with-mon-dashboard.adoc

```diff
@@ -28,29 +31,29 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. From the *Developer* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+. In the *Developer* perspective of the {product-title} web console, go to *Observe* -> *Metrics*.
 
-. Select the project that you want to view metrics for in the *Project:* list.
+. Select the project that you want to view metrics for from the *Project:* list.
 
-. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*. The metrics from the queries are visualized on the plot.
+. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by clicking *Show PromQL*. The metrics from the queries are visualized on the plot.
 +
 [NOTE]
 ====
-In the Developer perspective, you can only run one query at a time.
+In the *Developer* perspective, you can only run one query at a time.
```

Comment:
see above picture, 4.17+ supports run multiple queries

Reply:
Yes, thank you for the information, I will note it in that new Jira I am creating for this :+1: